### DISCUSSION TITLE
Run Pluto.jl in debug mode?

### DISCUSSION THREAD

**Discussion by samuela** [fonsp/Pluto.jl#1395] [Pluto development] [answered]
Is there any way to get Pluto to log debug messages? I'm interested in tracking down the cause of https://github.com/fonsp/Pluto.jl/issues/1370 in particular.

**Comment by fonsp** [comment, marked as answer]
This is an issue with HTTP, and I am sure that they have a debug mode, you probably activate it with `ENV["JULIA_DEBUG"] = Pluto.HTTP`. https://docs.julialang.org/en/v1/stdlib/Logging/#Environment-variables

**Reply by fonsp** [reply]
Pluto does not have many `@debug` messages, most of our development happens on the JS side, see https://github.com/fonsp/pluto-developer-instructions for more info

**Reply by samuela** [reply]
Is there an upstream issue in the HTTP repo for this?
